From 33c06b169149a5acc91d8e5d2edd6483d8e2c7d0 Mon Sep 17 00:00:00 2001
From: Robert Marshall <rmarshall@redhat.com>
Date: Wed, 21 Jan 2015 15:51:12 -0500
Subject: [PATCH 27/41] Update grubby man page for Power8 PPC64LE

The grubby man page was up to date for Power7 PPC systems using
yaboot. Added the information for the grub2-based Power8 PPC64LE
system architecture.

Resolves bz#1125540
---
 grubby.8 | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/grubby.8 b/grubby.8
index 9d7a1fd..5ff486f 100644
--- a/grubby.8
+++ b/grubby.8
@@ -28,8 +28,10 @@ the configuration file is in \fB/boot/grub2/grub.cfg\fR.  On UEFI-based Intel
 x86 platforms, \fBgrub2\fR is the default bootloader, and the configuration
 file is in \fB/boot/efi/EFI/redhat/grub.cfg\fR.  On Intel ia64 platforms,
 \fBelilo\fR mode is used and the default location for the configuration file
-is \fB/boot/efi/EFI/redhat/elilo.conf\fR. On PowerPC platforms, \fByaboot\fR
-parsing is used and the configuration file should be in \fB/etc/yaboot.conf\fR.
+is \fB/boot/efi/EFI/redhat/elilo.conf\fR. On PowerPC platforms, systems based
+on Power8 now support \fBgrub2\fR as a bootloader and store using a default
+config stored in \fB/boot/grub2/grub.cfg\fR. The earlier Power7 systems use \fByaboot\fR
+parsing and the configuration file should be in \fB/etc/yaboot.conf\fR.
 
 There are a number of ways to specify the kernel used for \fB-\-info\fR,
 \fB-\-remove-kernel\fR, and \fB-\-update-kernel\fR. Specificying \fBDEFAULT\fR
-- 
2.4.3

